samples/Samples.h

```diff
@@ -64,7 +64,8 @@ extern "C" {
 #define IOT_CORE_THING_NAME          ((PCHAR) "AWS_IOT_CORE_THING_NAME")
 #define IOT_CORE_CERTIFICATE_ID      ((PCHAR) "AWS_IOT_CORE_CERTIFICATE_ID")
 
-/* Uncomment the following line in order to enable IoT credentials checks in the provided samples */
+/* Uncomment the following line in order to enable IoT credentials checks in the provided samples
+   Or, you pass specify this through the CMake flag: cmake .. -DIOT_CORE_ENABLE_CREDENTIALS=ON */
 // #define IOT_CORE_ENABLE_CREDENTIALS  1
```

Comment:
can be removed in subsequent

Reply:
thanks for the confirmation, let's remove it in this because it's redundant